src/cvodes/cvodea_io.c

```diff
@@ -174,6 +174,92 @@ int CVodeSetUserDataB(void* cvode_mem, int which, void* user_dataB)
   return (CV_SUCCESS);
 }
 
+int CVodeSetOwnUserDataB(void* cvode_mem, int which, sunbooleantype own_user_data)
```

Comment:
Prefix internal function with `cv`
```suggestion
int cvSetOwnUserDataB(void* cvode_mem, int which, sunbooleantype own_user_data)
```